### PR TITLE
Develop to main

### PR DESCRIPTION
**Pull request by dcolleoni**
Release v1.1.0

### CODE REVIEW

File: ifrs17/Import/Importers.ipynb [hunk 1]
```diff
@@ -135,10 +138,8 @@
                 "\n                ImportFormats.Opening => aocConfigurationByAocStep.Where(x => x.InputSource.Contains(InputSource.Opening) && x.DataType == DataType.Optional).GroupBy(x => new AocStep(x.AocType, x.Novelty), (k,v) => k).ToHashSet(),",
                 "\n                ImportFormats.SimpleValue => aocConfigurationByAocStep.GroupBy(x => new AocStep(x.AocType, x.Novelty), (k,v) => k).Concat((await dataSource.Query<PnlVariableType>().ToArrayAsync())",
                 "\n                                                                                   .Select(vt => new AocStep(vt.SystemName,null))).ToHashSet(),",
-                "\n                _ => Enumerable.Empty<AocStep>().ToHashSet(),",
+                "\n                 _ => Enumerable.Empty<AocStep>().ToHashSet(),",
```
Comment (nnikolopoulos) [inline ifrs17/Import/Importers.ipynb:141]: Unnecessary space (minor comment)

File: ifrs17/Constants/Validations.ipynb [hunk 2]
```diff
@@ -184,7 +184,9 @@
                 "public static string Get (Warning w, params string[] s) => (w, s.Length) switch {",
                 "\n    // Import",
                 "\n    (Warning.ActiveDataNodeWithCashflowBOPI , 1) => $\"Cash flow with AoC Type: {AocTypes.BOP} and Novelty: {Novelties.I} for Group of Contract {s[0]} is not allowed because previous period data are available.\",",
-                "\n    (Warning.ScenarioReCalculations         , 1) => $\"The import of the current file for the Best Estimate scenario makes the result of dependent Scenarios out of date. Hence, the following Scenarios are recalculated: {s[0]}.\", ",
+                "\n    (Warning.VariablesAlreadyImported       , 0) => $\"The import of the current file does not contain any new data. Hence, no data will be saved or calculations will be performed.\",",
```
Comment (nnikolopoulos) [inline ifrs17/Constants/Validations.ipynb:187]: Typo? "... be saved or calculations ..." -->  "... be saved nor calculations ..."

**Review by nnikolopoulos — APPROVED** [review]

File: ifrs17/Import/Importers.ipynb (hunk 1 above)
Comment (dcolleoni) [reply ifrs17/Import/Importers.ipynb:141]: minor

File: ifrs17/Constants/Validations.ipynb (hunk 2 above)
Comment (dcolleoni) [reply ifrs17/Constants/Validations.ipynb:187]: Minor. We can capture this in an issue and tackle it in the next release.

**Pull request merged**